sound/soc/sof/sof-client-ipc-flood-test.c

```diff
@@ -219,9 +219,10 @@ static ssize_t sof_ipc_flood_dfs_write(struct file *file, const char __user *buf
 		goto out;
```

Comment:
bad language in commit, "with introduction of on-demand DSP boot, the rpm status does not necessarily tell that the DSP firmware has booted up"